docs/sdk/client-side-sdks/android.md

```diff
@@ -46,10 +46,11 @@ override fun onCreate(savedInstanceState: Bundle?) {
         )
         .withEnvironmentKey("<DEVCYCLE_MOBILE_ENVIRONMENT_KEY>")
         .build()
-    })
+    }
```

Comment:
I think this closing tag is also unnecessary

Reply:
Nope that one necessary, see the highlighted associated tag: 

<img width="655" alt="image" src="https://user-images.githubusercontent.com/1301285/170331078-a534b96f-9867-449b-8a11-c7f1bd867596.png">